loadtests/prep-maven-dyn.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+"""
+Configurable maven test data generator
+Usage::
+    ./gen-maven.py groups artifacts versions big_size med_size small_size bigs_percent smalls_percent
+"""
+
+from io import TextIOBase
+import os
+import shutil
+import pathlib
+import signal
+import sys
+import tempfile
+import subprocess
+import time
+
+g_termination = False
+
+def generateRepo(dstRepo: str, srv: subprocess.Popen, groups, artifacts, versions, bigSize, medSize, smallSize, bigP, smallP):
+    BIG = 'big'
+    MED = 'med'
+    SMALL = 'small'
+
+    totals = int(groups * artifacts * versions)
+    bigs = int(totals * bigP / 100)
+    smalls = int(totals * smallP / 100)
+    mediums = int(totals - bigs - smalls)
+    if totals <= 0 or mediums < 0:
+        print(f"Error. Incorrent artifacts counts: {totals}; {bigs}; {mediums}; {smalls}", file=sys.stderr)
+        exit(1)
+
+    print(f"Limits: {totals}; {bigs}; {smalls}, {mediums}")
```

Comment:
Add info for every digits. Like a big=20, medium=100 and etc.
And order should be big, medium, small.